playbooks/common/openshift-cluster/openshift_hosted.yml

```diff
@@ -57,6 +57,10 @@
     openshift_hosted_logging_master_public_url: "{{ logging_master_public_url }}"
     openshift_hosted_logging_elasticsearch_cluster_size: "{{ logging_elasticsearch_cluster_size }}"
     openshift_hosted_logging_elasticsearch_ops_cluster_size: "{{ logging_elasticsearch_ops_cluster_size }}"
+    openshift_hosted_logging_elasticsearch_pvc_dynamic: "{{ 'true' if openshift.hosted.logging.storage_kind | default(none) == 'dynamic' else 'false' }}"
+    openshift_hosted_logging_elasticsearch_pvc_size: "{{ openshift.hosted.logging.storage.volume.size if openshift.hosted.logging.storage_kind | default(none) == 'dynamic' else ''  }}"
+    openshift_hosted_logging_elasticsearch_pvc_prefix: "{{ 'logging-es' if openshift.hosted.logging.storage_kind | default(none) is not none else '' }}"
```

Comment:
Do you need corresponding `ops` parameters for the ops cluster e.g. `openshift_hosted_logging_elasticsearch_ops_pvc_dynamic` ?